backend/learnings/views.py

```diff
@@ -126,3 +126,13 @@ def create(self, request):
                 is_correct=check,
             )
         return Response({"quiz_taken_id": get_quiz_taken_instance.id})
+
+
+class GetResults(APIView):
+    permission_classes = [IsAuthenticated]
+    authentication_classes = [TokenAuthentication]
+
+    def get(self, request, quiz_taken_id):
+        get_user_answers = UserAnswer.objects.filter(quiz_taken_id_id=quiz_taken_id)
```

Comment:
is this filter working?
this looks off `quiz_taken_id_id=quiz_taken_id`

Reply:
Yes po its working

Reply:
Sir I think tama lang sya kasi yung old code ay 

```
get_quiz_taken_instance = QuizTaken.objects.get(id=quiz_taken_id)
get_user_answers = UserAnswer.objects.filter(quiz_taken_id=get_quiz_taken_instance.id)
```

Ang ginawa ko jan ay kinuha ko yung instance ng taken id then yung instance mismo ang ginamit pag filter so bali object by object sya peroooo ngayon kay hindi na, yung id na mismo sa fk yung ginamit ko pag compare

Reply:
hmmm i was thinking something like `quiz_taken_id__id` instead of `quiz_taken_id_id` , but since it is working then i guess im thinking wrong ahaha :bow: